docs/UserGuide.adoc

```diff
@@ -369,43 +369,60 @@ Displays all persons whose tissue type contains tissues: 4, 1, 2 and 3; in any o
 
 === Detecting matches: `match`
 
-This command runs a kidney matching test on patients and donors in ORGANice and displays the match results.
+This command runs a kidney matching test on patients and donors in ORGANice and displays potential matches.
```

Comment:
Maybe say `organ` instead of `kidney`? cause we are actually organ matching thingy?

Reply:
I think I should indicate that it is kidney matching. Because this match command is coupled with the 'processing' and 'done' command, which is developed specifically with kidney matching in mind. 

However, I can indicate that if support for more organs are added, this match command will match based on the organ specified.